themes/V3/Blank/style.less

```diff
@@ -167,6 +167,103 @@ body {
 	.watercolor11 { --wc : @watercolor11; }
 	.watercolor12 { --wc : @watercolor12; }
 
+	/* Image Masks */
+	[class*="imageMask"] {
+		position              : absolute;
+		height                : 200%;
+		width                 : 200%;
+		left                  : 50%;
+		bottom                : 50%;
+		--rotation            : 0;
```

Comment:
When using `--rotation` with non-cardinal degrees I'm getting an interesting visual issue. Is there a way to resolve this I've missed?

![image](https://user-images.githubusercontent.com/4909307/211131002-88f0d0c3-c639-46a5-b8af-9b50a59d6b2a.png)

Reply:
They should be tiled horizontally (designed to be seamless) so they extend infinitely to the sides. It seems this isn't kicking in anymore.... I'll investigate...